cache-config/testing/ort-tests/t3c_mode_test.go

```diff
@@ -132,43 +171,7 @@ func TestT3cBadassAndSyncDs(t *testing.T) {
 
 		time.Sleep(time.Second * 5)
 
-		fmt.Println("------------------------ Verify Plugin Configs ----------------")
-		err = runCheckRefs("/opt/trafficserver/etc/trafficserver/remap.config")
-		if err != nil {
-			t.Errorf("Plugin verification failed for remap.config: " + err.Error())
-		}
-		err = runCheckRefs("/opt/trafficserver/etc/trafficserver/plugin.config")
-		if err != nil {
-			t.Errorf("Plugin verification failed for plugin.config: " + err.Error())
-		}
-
-		fmt.Println("----------------- End of Verify Plugin Configs ----------------")
-
-		fmt.Println("------------------------ running SYNCDS Test ------------------")
-		// remove the remap.config in preparation for running syncds
-		remap := test_config_dir + "/remap.config"
-		err = os.Remove(remap)
-		if err != nil {
-			t.Fatalf("ERROR: unable to remove %s\n", remap)
-		}
-		// prepare for running syncds.
-		err = ExecTOUpdater("atlanta-edge-03", false, true)
-		if err != nil {
-			t.Fatalf("ERROR: queue updates failed: %v\n", err)
-		}
-
-		// remap.config is removed and atlanta-edge-03 should have
-		// queue updates enabled.  run t3c to verify a new remap.config
-		// is pulled down.
-		err = runApply("atlanta-edge-03", "syncds")
-		if err != nil {
-			t.Fatalf("ERROR: t3c syncds failed: %v\n", err)
-		}
-		if !util.FileExists(remap) {
-			t.Fatalf("ERROR: syncds failed to pull down %s\n", remap)
-		}
-		fmt.Println("------------------------ end SYNCDS Test ------------------")
-
+		t.Run("Verify Plugin Configs", verifyPluginConfigs)
+		t.Run("SyncDS Test", syncDSTest)
```

Comment:
`syncDSTest` doesn't really explain what the test is doing here.
It's deleting remap.config, and verifying a t3c run places a new one. Maybe something like "verifyRemapConfigPlaced" ?